drivers/shared/executor/executor_linux.go

```diff
@@ -805,52 +788,125 @@ func cmdMounts(mounts []*drivers.MountConfig) []*lconfigs.Mount {
 	return r
 }
 
-// lookupTaskBin finds the file `bin` in taskDir/local, taskDir in that order, then performs
-// a PATH search inside taskDir. It returns an absolute path. See also executor.lookupBin
-func lookupTaskBin(command *ExecCommand) (string, error) {
+// lookupTaskBin finds the file `bin` in taskDir/local, taskDir in that order,
+// then performs a PATH search inside taskDir. It returns an absolute path
```

Comment:
needs a mention of the mount paths in the ordering - might be nice to format this as a bullet list, including the PATH style lookup dirs

Reply:
and the search order of the mounts is the order of mounts listed on the task, correct?

Reply:
might be good to detail all this on the exec docs page

Reply:
Agreed, will do.

Reply:
Just like I imagined!